src/Kovan-DssSpell.t.sol

```diff
@@ -183,7 +148,7 @@ contract DssSpellTest is DSTest, DSMath {
     // Rates table is in ./test/rates.sol
```

Comment:
I think `end_old` and `esm_old` on line 116, 117 can be removed.

Reply:
Fixed